src/main/java/VASSAL/build/module/GlobalOptions.java

```diff
@@ -455,6 +476,10 @@ public boolean centerOnOpponentsMove() {
   public boolean isMarkMoveEnabled() {
     return isEnabled(markMoved, MARK_MOVED);
   }
+  
+  public static int getDragThreshold() {
```

Comment:
What's your reason for making this static? (The other members which are set from stored values have getters which aren't.)

Reply:
Oh right. I had to try to change it back to remember why -- it's because the place I need access to it in GameModule is static.

Reply:
Can't you make this non-static and get at it via `GlobalOptions.getInstance()`?

Reply:
Uh, why apparently yes, he says sheepishly.  (A pattern that seems common here in java that I don't remember seeing much in my good ole C++ days)

Done, checked in.

Reply:
Singletons were common in terrible crappy old Java.